modules/c++/cli/unittests/test_cli.cpp

```diff
@@ -175,7 +175,7 @@ TEST_CASE(testRequired)
     TEST_ASSERT_EQ(results->get<std::string>("config"), "configFile");
 }
 
-int main(int argc, char* argv[])
+int main()
```

Comment:
While this compiles, I'd prefer the normal two argument signature, so how about
`int main(int , argc )`